docs/getting-started.md

```diff
@@ -0,0 +1,347 @@
+
+__Note:__ The service will NOT be accessible until you create an OpenShift route with `oc expose`. You must also have a virtualization environment setup. By default, `minishift` uses VirtualBox.
+
+__Start `minishift`:__
+
+[Minishift](https://github.com/minishift/minishift) is a tool that helps run OpenShift locally using a single-node cluster inside of a VM. Similar to [minikube](https://github.com/kubernetes/minikube].
+
+```sh
+$ minishift start
+Starting local OpenShift cluster using 'kvm' hypervisor...
+-- Checking OpenShift client ... OK
+-- Checking Docker client ... OK
+-- Checking Docker version ... OK
+-- Checking for existing OpenShift container ... OK
+...
+```
+
+__Download an [example Docker Compose file](https://raw.githubusercontent.com/kubernetes/kompose/master/examples/docker-compose.yaml), or use your own:__
+
+```sh
+wget https://raw.githubusercontent.com/kubernetes/kompose/master/examples/docker-compose.yaml
+```
+
+__Convert your Docker Compose file to OpenShift:__
+
+Run `kompose convert --provider=openshift` in the same directory as your `docker-compose.yaml` file.
+
+```sh
+$ kompose convert --provider=openshift
+INFO OpenShift file "frontend-service.yaml" created 
+INFO OpenShift file "redis-master-service.yaml" created 
+INFO OpenShift file "redis-slave-service.yaml" created 
+INFO OpenShift file "frontend-deploymentconfig.yaml" created 
+INFO OpenShift file "frontend-imagestream.yaml" created 
+INFO OpenShift file "redis-master-deploymentconfig.yaml" created 
+INFO OpenShift file "redis-master-imagestream.yaml" created 
+INFO OpenShift file "redis-slave-deploymentconfig.yaml" created 
+INFO OpenShift file "redis-slave-imagestream.yaml" created 
+```
+
+Alternatively, you can convert and deploy directly to OpenShift with `kompose up --provider=openshift`.
+
+```sh
+$ kompose up --provider=openshift       
+INFO We are going to create OpenShift DeploymentConfigs, Services and PersistentVolumeClaims for your Dockerized application. 
+If you need different kind of resources, use the 'kompose convert' and 'oc create -f' commands instead. 
+ 
+INFO Deploying application in "myproject" namespace 
+INFO Successfully created Service: frontend       
+INFO Successfully created Service: redis-master   
+INFO Successfully created Service: redis-slave    
+INFO Successfully created DeploymentConfig: frontend 
+INFO Successfully created ImageStream: frontend   
+INFO Successfully created DeploymentConfig: redis-master 
+INFO Successfully created ImageStream: redis-master 
+INFO Successfully created DeploymentConfig: redis-slave 
+INFO Successfully created ImageStream: redis-slave 
+
+Your application has been deployed to OpenShift. You can run 'oc get dc,svc,is,pvc' for details.
+```
+
+__Access the newly deployed service:__
+
+After deployment, you must create an OpenShift route in order to access the service.
+
+If you're using `minikube`, you'll use a combination of `oc` and `minishift` commands to access the service.
```

Comment:
replace `minikube` with `minishift`

Reply:
@cdrage ^^

Reply:
@surajnarwade GitHub is messing up. Ctrl+F5 and refresh the page and you should see that I've already replaced it.